docs/1-Using-Fleet/configuration-files/README.md

```diff
@@ -351,6 +350,36 @@ spec:
                 - "last_modified"
 ```
 
+#### YARA configuration
+
+You can use Fleet to configure the `yara` and `yara_events` osquery tables. Fore more information on YARA configuration and continuous monitoring using the `yara_events` table, check out the [YARA-based scanning with osquery section](https://osquery.readthedocs.io/en/stable/deployment/yara/) of the osquery documentation.
+
+The following is an example Fleet configuration file with YARA configuration. The values are taken from an example config supplied in the above link to the osquery documentation.
+
+```yaml
+---
+apiVersion: v1
+kind: config
+spec:
+  agent_options:
+    config:
+      ...
```

Comment:
```suggestion
      # ...
```

(Doing this would make the example valid yaml)

Reply:
Awesome. Thank you. Adding this treatment to a couple other examples.